### PR TITLE
Widget Visibility: use translation function for taxonomy rules.

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

File: modules/widget-visibility/widget-conditions.php [hunk 1]
```diff
@@ -156,7 +156,7 @@ public static function widget_conditions_options_echo( $major = '', $minor = ''
 				foreach ( $taxonomies as $taxonomy ) {
 					?>
 					<optgroup label="<?php esc_attr_e( $taxonomy->labels->name . ':', 'jetpack' ); ?>">
-						<option value="<?php echo esc_attr( $taxonomy->name ); ?>" <?php selected( $taxonomy->name, $minor ); ?>><?php echo 'All ' . esc_html( $taxonomy->name ) . ' pages'; ?></option>
+						<option value="<?php echo esc_attr( $taxonomy->name ); ?>" <?php selected( $taxonomy->name, $minor ); ?>><?php printf( _x( 'All %s pages', 'Taxonomy name', 'jetpack' ), esc_html( $taxonomy->name ) ); ?></option>
```
Comment (zinigor) [inline modules/widget-visibility/widget-conditions.php]: Looks good, but can you please use a translator comment here instead of context?

**Comment by zinigor** [comment]
Thanks! Looks good except one minor thing.

**Comment by jeherve** [comment]
@zinigor Updated.

**Comment by dereksmart** [comment]
ty @jeherve !

**Pull request merged**